src/mongo/db/storage/mmap_v1/file_allocator.cpp

```diff
@@ -472,11 +472,9 @@ void FileAllocator::run(FileAllocator* fa) {
     }
 }
 
-FileAllocator* FileAllocator::_instance = 0;
+FileAllocator* FileAllocator::_instance = new FileAllocator();
```

Comment:
Would you mind putting up a pull request to that effect, and I'll take care of merging it?